tpm2/structures.go

```diff
@@ -371,3 +371,158 @@ type tpmtSigScheme struct {
 	Scheme Algorithm
 	Hash   Algorithm
 }
+
+// AttestationData contains data attested by TPM commands (like Certify).
+type AttestationData struct {
+	Magic               uint32
+	Type                tpmutil.Tag
+	QualifiedSigner     Name
+	ExtraData           []byte
+	ClockInfo           ClockInfo
+	FirmwareVersion     uint64
+	AttestedCertifyInfo *CertifyInfo
+}
+
+// DecodeAttestationData decode a TPMS_ATTEST message. No error is returned if
+// in has extra trailing data.
```

Comment:
I originally thought that "in has trailing data" was a mistake, but then I saw that the name of the input argument was `in`. That suggests that while the comment is technically correct, it could use revision for clarity. How about "No error is return if the input has extra trailing data"?

Reply:
Yep, that's nicer. Done